identity/app/views/consentJourney.scala.html

```diff
@@ -135,6 +137,20 @@ <h3 class="manage-account__switch-title"></h3>
     )
 }
 
+@introGdprCampaignStep= @{
+    ConsentInfo(
+        title = "Please select the emails you wish to continue receiving",
+        help = List(
+            ConsentStepHelpText(
+                s"Why are we asking you to re-subscribe? <a class='u-underline' data-link-name='gdpr-oi-campaign : consents : to-landing' href='${LinkTo("https://gu.com/staywithus")}'>Find out here</a>"
```

Comment:
maybe make this the full `www.theguardian.com`.  I'm not sure LinkTo understands gu.com (but it will still work either way I suspect).

Reply:
it does!